.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,58 @@
+name: Version Bump
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+    branches:
+      - production
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get version number from pull request title
+        id: check-release
+        if: startsWith(github.event.pull_request.title, 'Release') && github.event.review.state == 'APPROVED'
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+        run: |
+          new_version=$(echo "$PR_TITLE" | cut -d' ' -f2)
+          if [[ ! "$new_version" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Version number "$new_version" not found or invalid. Exiting."
+            exit 1
+          fi
+          echo "$new_version"
+          echo "new_version=$new_version" >> $GITHUB_OUTPUT
+
+      - name: Checkout source code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Bump version number
```

Comment:
Needs to put check-release as dependency for this one. Otherwise this step will run regardless of resulrs from check-release